scripts/zones/Grand_Palace_of_HuXzoi/mobs/Qnaern.lua

```diff
@@ -2,22 +2,18 @@
 -- Area: Grand Palace of HuXzoi
 --  MOB: Qn'aern
 -----------------------------------
-mixins = {require("scripts/mixins/job_special")};
-require("scripts/globals/status");
+mixins = {require("scripts/mixins/job_special")}
+require("scripts/globals/status")
 -----------------------------------
 
-function onMobInitialize(mob)
```

Comment:
Don't we need the function even if it's not doing anything?

Reply:
its `onMobDeath()` that does that